schema/PARTY_JF_TRANSFER_MEMO.json

```diff
@@ -0,0 +1,445 @@
+{
+    "$schema": "https://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/fecgov/fecfile-validate/blob/main/schema/PARTY_JF_TRANSFER_MEMO.json",
+    "version": "v0.0.0.0",
+    "title": "FEC Party JF Memo",
+    "description": "Party JF Memo (12)",
+    "type": "object",
+    "required": [
+        "form_type",
+        "filer_committee_id_number",
+        "transaction_type_identifier",
+        "transaction_id",
+        "entity_type",
+        "contributor_organization_name",
+        "contributor_street_1",
+        "contributor_city",
+        "contributor_state",
+        "contributor_zip",
+        "contribution_date",
+        "contribution_amount",
+        "contribution_aggregate",
+        "aggregation_group",
+        "contribution_purpose_descrip",
+        "memo_code"
```

Comment:
missing `donor_committee_fec_id` and `donor_committee_name`

Reply:
nice catch.  Fixed in latest push.